spectacles/client.py

```diff
@@ -840,6 +840,29 @@ def lookml_validation(self, project) -> JsonDict:
         result = response.json()
         return result
 
+    def cached_lookml_validation(self, project) -> Optional[JsonDict]:
+        logger.debug(f"Getting cached LookML validation results for '{project}'")
+        url = utils.compose_url(self.api_url, path=["projects", project, "validate"])
+        response = self.get(url=url, timeout=TIMEOUT_SEC)
+
+        try:
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
+            raise LookerApiError(
+                name="unable-to-get-cached-lookml-validation",
+                title=f"Couldn't get cached LookML validation results in project {project}.",
+                status=response.status_code,
+                detail=(
+                    "Failed to get cached LookML valiation results. Please try again."
```

Comment:
If there are any error cases we do understand, would be good to elaborate on them in the error message, like "Cache results may have expired, please re-run the LookML validator to populate the cache."

Reply:
There aren't any cases that I'm aware of that cause this error to be triggered. If there is no cached result, it returns a 204 instead of a 4XX.